Lib/test/test_abc.py

```diff
@@ -448,15 +448,16 @@ class S(metaclass=abc_ABCMeta):
 
             # Also check that issubclass() propagates exceptions raised by
             # __subclasses__.
+            class CustomError(Exception): ...
```

Comment:
I overlooked this on the initial PR, but the preference is to use `pass` for an empty body (I realize this differs to how `.pyi` files typically do things.